settings.gradle

```diff
@@ -0,0 +1,17 @@
+pluginManagement {
+    repositories {
```

Comment:
Are any of these repositories used ?

Reply:
The block `pluginManagement` as the name says is required to provide sources for the Gradle plugins. Without this block Gradle doesn't know where to get e.g. the Android Gradle Plugin, so it is required. It is possible to remove individual repositories, but you never know about dependencies of Gradle plugins and to reduce headaches in the future, I would prefer to leave those repositories as they are. Just Jitpack could be useless.

The block `dependencyResolutionManagement` is more interesting. While the block `pluginManagement` specifies the repositories for the Gradle plugins, this block specifies the repositories for dependencies specified in the `dependencies` block in the build.gradle file. Since there are no dependencies right now, it is currently safe to remove. But adding any additional dependencies in the future will fail because Gradle doesn't know where to search for them.

Reply:
It's fine if dependency resolution fail when adding. It should be easy to add the required repositories.

Reply:
I noticed during testing that when removed the build fails due to unmet dependencies. It seems like the block `dependencyResolutionManagement` is also required to manage the dependencies of the Gradle plugins. I can safely remove Jitpack, but besides that and Maven Central for the `pluginManagement` section, every other repository seems mandatory.

Reply:
Ok, thanks for trying out.